### PR TITLE
Add a contributors json

### PR DESCRIPTION
**Pull request by rzkytmgr**

### CODE REVIEW

File: src/contributors/rzkytmgr.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+        "name": "Sabharatu Punten",
+        "desc": "Backend Developer",
+        "github": "https://github.com/rzkytmgr",
+        "twitter": "",
```
Comment (iamdarshshah) [inline src/contributors/rzkytmgr.json:5]:
```suggestion
        
```
Don't push the `null` param. Remove it, if you don't have the link

File: src/contributors/rzkytmgr.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+        "linkedin": ""
```
Comment (iamdarshshah) [inline src/contributors/rzkytmgr.json:6]:
```suggestion
       
```
Don't push the `null` param. Remove it, if you don't have the link

**Comment by rzkytmgr** [comment]
Done, i removed it

**Review by iamdarshshah — APPROVED** [review]
LGTM! Thanks, @rzkytmgr

**Pull request merged**